app/controllers/provider_interface/offer/conditions_controller.rb

```diff
@@ -38,9 +38,15 @@ def update
     private
 
       def conditions_params
-        params.require(:provider_interface_offer_wizard).permit(:further_condition_1, :further_condition_2,
-                                                                :further_condition_3, :further_condition_4,
-                                                                standard_conditions: [])
+        conditions_params = params.require(:provider_interface_offer_wizard)
```

Comment:
this method is doing too much, can this be split up into multiple methods, one to permit params and one to calculate the `further_conditions`?